### ISSUE TITLE
How to keep the learned parameters not far away from the certain parameters?

### ISSUE THREAD

**Issue by danieljf24** [fchollet/keras#1424]
In keras, it implemented the regularization technique. For example, if I'd like to implement loss function as `Loss_function + a||W||_2`, I just add a code `<model.add(Dense(64, input_dim=64, W_regularizer=l2(0.01))>` to make it.

But now, I'd like to implement `Loss_function + a||W-W_0||_2`， hopping the learned parameter  `W` not far away from  fixed parameter `W_0`. I do not know how to implement it with keras.
Any help is greatly appreciated. Thanks.

**Comment by fchollet** [comment]
You should implement your own regularizer --take a look at `regularizers.py`.

**Comment by danieljf24** [comment]
@fchollet Thanks for your advice!
